README.md

```diff
@@ -1,13 +1,48 @@
 # Visual Studio Project System Extensibility Documentation
 
+## What is a project system?
+A project system sits between a project file on disk (for example, .csproj and .vbproj) and various Visual Studio features including, but not limited to, Solution Explorer, designers, the debugger, language services, build and deployment. Almost all interaction that occurs with files contained in a project file, happens through the project system.
+
+There are three reasons to extend a project system in Visual Studio:
+1. Support a new project file format.
+1. Integrate existing file format with a new language service.
+1. Customize behavior of an existing project system.
+
+## Project system extensibility
+The traditional way to a build or customize a project system  is to implement a set of [Project System COM interfaces](https://docs.microsoft.com/en-us/visualstudio/extensibility/creating-a-basic-project-system-part-1). Most project systems don't start from scratch. Instead, they leverage the [MPFProj project system example](https://docs.microsoft.com/en-us/visualstudio/extensibility/internals/using-the-managed-package-framework-to-implement-a-project-type-csharp?view=vs-2017) as a starting point.
+
+The Visual Studio Project System (VSPS) described in this repository provides default implementation for a subset of project system COM interfaces. This simplifies building and maintaining a project system, but comes with the cost of reduced functionality.
+
+## How to select a project system platform?
+|Scenario|Recommended Project System Platform
+|---|---
+|Customize C#/VB/F# Desktop project system| MPFProj and COM-based [project flavoring](https://docs.microsoft.com/en-us/visualstudio/extensibility/internals/project-types?view=vs-2017)
```

Comment:
I don't think we want to recommend people to use MPF to customize traditional C#/VB project systems.  MPF is largely used to create a new project system, instead of doing customization.  I think MPF is being phasing out (not sure anyone is supporting/improving that anymore.